docs/index.rst

```diff
@@ -76,8 +76,9 @@ Client example::
             html = await fetch(session, 'http://python.org')
             print(html)
 
-    loop = asyncio.get_event_loop()
-    loop.run_until_complete(main())
+    if __name__ == '__main__':
+        loop = asyncio.get_event_loop()
+        loop.run_until_complete(main())
```

Comment:
maybe py3.7-style `asyncio.run()`?

Reply:
Not yet.
I don't want to reply on multiple nubies complains "why the very first aiohttp example doesn't work on my laptop".

Reply:
agreed.